### PR TITLE
Protocol Description in Protocol Management becomes rich text field and is more exposed in U I [SCI-3073]

### PR DESCRIPTION
**Pull request by mz3944**
Jira ticket: [SCI-SCI-3073](https://biosistemika.atlassian.net/browse/SCI-SCI-3073)

### What was done
_Implemented loading of recently used repository protocols into task, and changed protocol header._

### Note:
_Buttons for the completing and locking of the task were removed, since they'll be moved up, above task details. I didn't leave them in, since it would ruin the expected UI. So this PR should be merged together when the part above protocol task is finished._

### CODE REVIEW

**Comment by okriuchykhin** [comment]
I think it should go to features/note-section and not master

File: app/controllers/my_modules_controller.rb [hunk 1]
```diff
@@ -256,6 +256,23 @@ def update
 
   def protocols
     @protocol = @my_module.protocol
+    @recent_protocols = Protocol.where(
+      team: current_team,
+      protocol_type: Protocol.protocol_types[:in_repository_public]
+    ).or(
+      Protocol.where(
```
Comment (okriuchykhin) [inline app/controllers/my_modules_controller.rb]: Do we allow loading protocols from other teams?

File: app/views/my_modules/protocols/_protocol_buttons.html.erb [hunk 2]
```diff
@@ -26,18 +26,24 @@
...
+      <a href="#" title="<%=t "my_modules.protocols.buttons.load_protocol_from_recent_repository" %>" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        <span class="hidden-sm hidden-md"><%=t "my_modules.protocols.buttons.load_protocol_from_recent_repository" %></span>
+        <span class="hidden-xs hidden-lg fas fa-list-alt"></span>
+        <span class="caret"></span>
+      </a>
+      <ul class="dropdown-menu pull-right">
+        <% @recent_protocols.each do |repo_protocol| %>
+          <li>
+            <%= link_to load_from_repository_protocol_path(@protocol, source_id: repo_protocol.id, format: :json), method: :post, remote: true, data: { action: "load-recent-from-repository" }, title: t('my_modules.protocols.buttons.load_protocol_from_recent_repository_title', protocol_name: repo_protocol.name) do %>
+              <i class="fas fa-file-alt"></i><%= truncate(repo_protocol.name) %>
```
Comment (okriuchykhin) [inline app/views/my_modules/protocols/_protocol_buttons.html.erb]: I think it should have grey color

File: app/controllers/my_modules_controller.rb (hunk 1 above)
Comment (mz3944) [reply app/controllers/my_modules_controller.rb]: Well I assumed we do, if user has access to them. @radiokills ?

**Pull request closed without merging**